internal/auth/authenticator.go

```diff
@@ -129,9 +127,14 @@ func New(client *http.Client, clientID, clientSecret string, opts ...Authenticat
 
 // VerifyDevice performs the device verification API calls.
 func (d *DeviceAuthenticator) VerifyDevice(ctx context.Context) (*DeviceVerification, error) {
-	oauthScopes := []string{"read_databases", "write_databases", "read_user", "read_organization"}
-	payload := strings.NewReader(fmt.Sprintf("client_id=%s&scope=%s", d.ClientID, url.PathEscape(strings.Join(oauthScopes, " "))))
-	req, err := d.NewFormRequest(ctx, http.MethodPost, "oauth/authorize_device", payload)
+	req, err := d.newFormRequest(ctx, "oauth/authorize_device", url.Values{
+		"client_id": []string{d.ClientID},
+		// Scopes are concatenated with strings and ultimately URL-encoded for
+		// use by the server.
```

Comment:
https://datatracker.ietf.org/doc/html/rfc6749#section-3.3 for if you want to throw in an RFC definition :wink:.